test/unit/model/define.test.js

```diff
@@ -163,7 +245,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
             type: DataTypes.MY_UNKNOWN_TYPE,
           },
         });
-      }).to.throw('Unrecognized datatype for attribute "bar.name"');
+      }).to.throw('Attribute "bar.name" does not specify its DataType.');
```

Comment:
it's more likely that someone will forget to specify the datatype than make a typo in their datatype name, so this error makes more sense I think